optional-dependencies.txt

```diff
@@ -4,8 +4,9 @@ face_recognition
 python-twitter
 gensim
 google-api-python-client
+google-compute-engine
```

Comment:
Just curious, where is this used?

Reply:
Nevermind, I see you just ported from the travis file.

Reply:
Yeah I wasn't actually sure it was used anywhere, but figured there was probably a reason it was in the travis file.